<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Todo::Internal::OptionsDialog</class>
 <widget class="QWidget" name="Todo::Internal::OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Keywords</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="keywordsList">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="addKeywordButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editKeywordButton">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeKeywordButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetKeywordsButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="scanningScopeGroupBox">
     <property name="title">
      <string>Scanning scope</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="scanInProjectRadioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Scan the whole active project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scanInCurrentFileRadioButton">
        <property name="text">
         <string>Scan only the currently edited document</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scanInSubprojectRadioButton">
        <property name="text">
         <string>Scan the current subproject</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>keywordsList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Todo::Internal::OptionsDialog</receiver>
   <slot>setKeywordsButtonsEnabled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setKeywordsButtonsEnabled()</slot>
 </slots>
</ui>
